mobile/library/common/jni/jni_interface.cc

```diff
@@ -1109,6 +1112,150 @@ Java_io_envoyproxy_envoymobile_engine_JniLibrary_registerStringAccessor(JNIEnv*
   return result;
 }
 
+//  Takes a jstring from Java, converts it to a C++ string, calls the supplied
+//  setter on it.
```

Comment:
Nit: single leading space after the `//`:

```suggestion
// Takes a jstring from Java, converts it to a C++ string, calls the supplied
// setter on it.
```